.travis.yml

```diff
@@ -2,7 +2,8 @@ language: java
 script:
   - set -e
   - export CROMWELL_VERSION=34
-  - wget https://github.com/broadinstitute/cromwell/releases/download/$CROMWELL_VERSION/womtool-$CROMWELL_VERSION.jar
-  - for F in *.wdl; do echo $F; java -jar womtool-$CROMWELL_VERSION.jar validate $F; done
+#  - wget https://github.com/broadinstitute/cromwell/releases/download/$CROMWELL_VERSION/womtool-$CROMWELL_VERSION.jar
+  - wget https://barmsijs.lumc.nl/womtool-35-a7ae2d8-SNAP.jar
```

Comment:
UGLY :scream: 
but the only option really until cromwell-35 is released

Reply:
agree, but there no other option for now